agent/lib/kontena/service_pods/creator.rb

```diff
@@ -159,6 +159,7 @@ def ensure_image(name)
       # @param [Docker::Container] service_container
       # @return [Boolean]
       def service_uptodate?(service_container)
+        return false unless service_container.running?
```

Comment:
Can this `return false` be reverted now that the `ServicePodWorker#ensure_running` invokes the `Kontena::ServicePods::Starter`?

The `Kontena::ServicePods::Creator` should never be called anymore if the container already exists and is up to date...